starter-code/css/main.css

```diff
@@ -6,3 +6,140 @@ Green: #00B172
 White: #FFF
 
 */
+
+nav{
+  top: 0px;
+  background-color: white;
+  position: fixed;
+  display: flex;
+  height: 80px;
+  padding-bottom: 50px;
+}
+
+.logo img{
+  max-width:30%;
+}
+
+.logo img:hover {
+  animation: bounce 1s;
+  animation-iteration-count: infinite;
+  animation-direction: alternate;
+  }
+
+.navText{
+  align-items: center;
+}
+
+ul{
```

Comment:
This selector works but wouldn't be specific enough in a real project. Use a class or a more specific selector so that your CSS rules don't affect other `ul`s.